spec/twilio_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'twilio.rb'
+
+describe TwilioSend do
+
+  it "should use Twilio to send a text message" do
+    expect(subject).to respond_to(:sms).with(1).argument
+  end
+
+  it 'should provide comfirmation message' do
+      time = (Time.now + 3600).strftime("%H:%M")
+      expect(subject.message).to eq(
+      "Thank you! Your order was placed and will be delivered before #{time}.")
+    end
```

Comment:
Lint/BlockAlignment: `end` at 13, 4 is not aligned with `it 'should provide comfirmation message' do` at 9, 2